storage/badger/operation/prefix.go

```diff
@@ -64,7 +64,6 @@ const (
 	codePayloadSeals      = 53 // index mapping block ID to payload seals
 	codeCollectionBlock   = 54 // index mapping collection ID to block ID
 	codeOwnBlockReceipt   = 55 // index mapping block ID to execution receipt ID for execution nodes
```

Comment:
```suggestion
	codeOwnBlockReceipt   = 55 // index mapping block ID to execution receipt ID for execution nodes
	// code 56 previously used for epoch status index, now deprecated
```
may be useful to leave a tombstone explaining the missing code

Reply:
What is the rational behind leaving a tombstone? Just out of curiosity.

My default would have been to decrement the previous values of the following each by 1: https://github.com/onflow/flow-go/blob/780880a99a977600a1bcc8196a2d1fc4b4fbb212/storage/badger/operation/prefix.go#L67-L69
Thereby, we close the gap. We have extended, shortened and replaced prefix conventions previously, with `prefix.go` only representing the latest version (and changes being incompatible). I wonder why we would do it differently now? I don't think it is tractable for us to keep a useful documentation of past usages for certain numerical values in this file.

Reply:
> decrement the values of the following prefixes each by 1

That works too -- mainly just didn't want to leave a gap with no explanation. I was hesitant to change the key prefixes, but you're right we've done that before, and don't have backward-compatibility anyway.

Reply:
I've updated it to be a continuous range